tests/test_wsireader.py

```diff
@@ -999,7 +999,7 @@ def test_VFReader_read_bounds():
 def test_VFReader_read_rect():
     """Test VFReader read bounds"""
```

Comment:
same as above "VirtualWSIReader"

Reply:
Addressed in this commit https://github.com/TIA-Lab/tiatoolbox/commit/8387736e723a92110cc719baf0490d46a6518d82